.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,57 @@
+--- # autorelease yml file for easybashgui
+
+name: auto release ebg # named it
+
+# Controls when the action will run
+on:  # will trigger
+
+  # Build at 00:00 on every 12th day-of-month.
+  #schedule:
+  #  - cron: "0 0 */12 * *"
+
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    #branches: '**' #'!master' # excludes master
+    #paths-ignore: [ '**/README.md' ]
+  #pull_request:
+    #branches: '**' # matches every branch
+    #paths-ignore: [ '**/README.md' ]
+
+    # Triggers the workflow if matching tag is created
+    tags: [ 'v*.*' ]
```

Comment:
now we must explain to @vaisarger  that will only trigerr when make a custom tag.. 

@vaisarger  it means no release will be publised .. and only if you tagged will be publised